src/prom-metrics.ts

```diff
@@ -93,4 +93,14 @@ export function configureSignerMetrics(db: PgStore) {
       }
     },
   });
+
+  new Gauge({
+    name: metricsPrefix + 'chain_tip_block_height',
```

Comment:
The stacks API's metric is `stacks_block_height`. Can we standardize on this since this metric is going to be in several API services? The `metricsPrefix` is fine, so `signer_api_stacks_block_height` should be the name I'd think.